snorkel/labeling/preprocess/nlp.py

```diff
@@ -1,12 +1,11 @@
-from typing import List
+from typing import List, Optional
 
 import spacy
 
 from snorkel.types import FieldMap
 
-from .core import Preprocessor
+from .core import BasePreprocessor, Preprocessor
 
-DEFAULT_DISABLE = ["parser", "ner"]
 EN_CORE_WEB_SM = "en_core_web_sm"
```

Comment:
use a more instructive variable name (e.g. `SPACY_CORPUS`?)

Reply:
going to leave this as is. don't want to bias people towards thinking this is a default spacy corpus (since it isn't, might want bigger or non-english model)

Reply:
why define a global variable in the first place, then?

Reply:
It's reused in NLPLabelingFunction